MAINTAINERS: add tdk team as TDK Sensors maintainers and collaborators

afontaine-invn opened the pull request:
We would like to take maintainership of TDK drivers as discussed:
 https://github.com/zephyrproject-rtos/zephyr/pull/85963

FYI @rbuisson-invn , @sriccardi-invn, @gjabouley-invn

nashif commented on MAINTAINERS.yml:
```diff
@@ -4448,6 +4448,11 @@ TDK Sensors:
   maintainers:
     - teburd
     - MaureenHelm
+    - afontaine-invn
```
do we still need to have @teburd and @MaureenHelm  here? Maybe move them to collaborators?

MaureenHelm replied:
Works for me. Thanks for stepping up @afontaine-invn !

afontaine-invn replied:
Ok, you are now moved as collaborators.

afontaine-invn committed:
MAINTAINERS: add tdk team as TDK Sensors maintainers and collaborators

We would like to take maintainership of TDK drivers as discussed:
 https://github.com/zephyrproject-rtos/zephyr/pull/85963
teburd and MaureenHelm moves to collaborators

Signed-off-by: Aurelie Fontaine <aurelie.fontaine@tdk.com>
